docs/external_authentication/README.md

```diff
@@ -0,0 +1,51 @@
+External Authentication Documentation
+============
+
+This README describes how to create a one-machine installation of all the components necessary for testing LDAP authentication in MongoDB.
+
+# Overview
+
+Normally, a client needs to authenticate themselves against the internal MongoDB user database before doing any work or reading any data from a `mongod` or `mongos` instance.  This project aims to allow the client to authenticate themselves through the MongoDB server to an independent service, such as OpenLDAP or Active Directory.
```

Comment:
That last sentence could be clearer, yes.  I am trying to express the fact that the actual authentication is occurring outside of the server, but all of the other aspects of the authentication, including imbuing the session with the user's associated authorizations, are still handled by the server.  I just don't have a concise way of expressing this idea...